app/scripts/modules/openstack/loadBalancer/configure/wizard/listeners.html

```diff
@@ -18,16 +17,14 @@
         <tr ng-repeat="listener in loadBalancer.listeners">
           <td><select class="form-control input-sm" ng-model="listener.externalProtocol"
                       ng-change="ctrl.listenerProtocolChanged(listener)"
-                      ng-options="protocol for protocol in ['HTTP','HTTPS','TCP','SSL']"></select></td>
+                      ng-options="protocol for protocol in ['HTTP','TERMINATED_HTTPS','TCP','SSL']"></select></td>
           <td><input class="form-control input-sm" type="number" min="0" ng-model="listener.externalPort"
                      required/></td>
           <td class="small" style="padding-top: 10px;">&rarr;</td>
-          <td><select class="form-control input-sm" ng-model="listener.internalProtocol"
-                      ng-options="protocol for protocol in ['HTTP','HTTPS','TCP','SSL']"></select></td>
           <td><input class="form-control input-sm" type="number" min="0" ng-model="listener.internalPort"
                      required/></td>
-          <td ng-if="ctrl.showSslCertificateIdField() && listener.externalProtocol !== 'HTTPS' && listener.externalProtocol !== 'SSL'"></td>
-          <td ng-if="listener.externalProtocol === 'HTTPS' || listener.externalProtocol === 'SSL'">
+          <td ng-if="ctrl.showSslCertificateIdField() && listener.externalProtocol !== 'TERMINATED_HTTPS' && listener.externalProtocol !== 'SSL'"></td>
```

Comment:
This can be consolidated if you have ng-if="ctrl.showSslCertificateIdField()" on "td" and ng-if="listener.externalProtocol === 'TERMINATED_HTTPS'" on the nested input.